include/seqan3/alignment/configuration/utility.hpp

```diff
@@ -66,7 +66,17 @@ namespace seqan3::align_cfg
 enum struct id : uint8_t
 {
     //!\brief Identifier for gap configuration.
-    gap
+    gap,
+    //!\brief Identifier for band configuration.
+    band,
+    //!\brief Identifier for free ends configuration.
+    free_ends,
+    //!\brief Identifier for score configuration.
+    score,
+    //!\brief Identifier for edit distance configuration.
+    edit,
```

Comment:
We do not need an extra field for edit. 
edit will be a combination of known variables already.

Reply:
Ok!